src/main/resources/messages/LanguageServerBundle.properties

```diff
@@ -33,10 +33,10 @@ language.server.mappings.fileNamePattern.no=No file name patterns mappings confi
 language.server.mappings.languageId.column=Language Id
 
 language.server.tab.configuration=Configuration
+language.server.tab.configuration.server=Server configuration

```

Comment:
These are the configuration sub-tab labels. Let me know if you'd prefer them to be changed.